app/src/main/java/com/example/sony/training/Regist.java

```diff
@@ -0,0 +1,51 @@
+package com.example.sony.training;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+public class Regist extends AppCompatActivity implements View.OnClickListener{
+    private EditText name,email,phone,password;
+    private Button register;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_regist);
+        name = (EditText) findViewById(R.id.name);
+        email = (EditText) findViewById(R.id.Email);
+        phone = (EditText) findViewById(R.id.Phone );
+        password = (EditText) findViewById(R.id.Password);
+        register = (Button) findViewById(R.id.Register);
+         register.setOnClickListener(this);
+    }
+
+    @Override
+    public void onClick(View v) {
+        final String ktemmail = "[A-Z0-9._%+-]+@[A-Z0-9-]+.+.[A-Z]{2,4}";
+        switch (v.getId()){
+            case R.id.btnLogin:
+                if (name.equals(null)&&email.equals(null)&&password.equals(null)){
+                    Toast.makeText(this,"Email , passs , name khong duoc de trong ",Toast.LENGTH_LONG).show();
+
+                }
+                if (!email.equals(ktemmail)&& password.length() <6){
+                    Toast.makeText(this,"Email khong dung dinh dang hoa pass nho hon 6 ky ",Toast.LENGTH_LONG).show();
+
+                }else {
+                    Toast.makeText(this,"dang ky thanh cong , chuyen ve trang dang nhap  ",Toast.LENGTH_LONG).show();
+                    Intent intent = new Intent(String.valueOf(login.class));
+                    startActivity(intent);
+                }
+
+                break;
+        }
+
+
+    }
+}
```

Comment:
change to
```java
String email = emailEditText.getText();
String userName = nameEditText.getText();
....
if(email.equals("email e cần check") && password.equals("password e cần check")) {
// Login thành công
}
```

Reply:
khonf kiem tra sao biet duoc gnguoi nhap mat khau vao , lo nguoi dung de trong thi sao